app/models/school.rb

```diff
@@ -16,6 +16,10 @@ class School < ApplicationRecord
   has_many :early_career_teacher_profiles
   has_many :early_career_teachers, through: :early_career_teacher_profiles, source: :user
 
+  scope :search_by_name_or_urn, lambda { |search_key|
```

Comment:
Is this specifically for searching by name and urn. I know for sure we'll have to search by other parameters like location etc. If we want it to be generic then the scope name probably needs generalising.

Reply:
We only want name and URN in the accessible autocomplete component though, right?

Reply:
Yes - we only have requirement for URN and name